content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md

```diff
@@ -0,0 +1,135 @@
+---
+date: 2017-09-12T15:20:00+02:00
+tags: [sales]
+title: Prediction of Huawei sales of Vidhance models
+author: Mr IMINT
+---
+
+# Prediction of Huawei sales of Vidhance models
+
+## Disclaimer
+Getting detailed information about sales volumes for specific Huawei models is hard. Therefore this analysis is based on the overall picture for smartphone sales from [Huawei Consumer Business Group][HCBG], combined with relevant official communications from Huawei about sales volumes for a few specific models, where this have been obtained. 
+
+If unfamiliar with Huawei, this [Fortune.com](http://fortune.com/huawei-china-smartphone/) article is a good start.
+
+## The bigger picture for Huawei smartphones
+
+![P8 Lite shipment trend](http://gulfnews.com/polopoly_fs/1.1979574.1487244664!/image/576257842.jpg "P8 Lite shipment trend")
+
+ * 2014: 75 million units shipped [(source: Huawei)](http://consumer.huawei.com/en/press/news/2017/hw-u-035103/)
+ * 2015: 108 million units shipped [(source: Huawei)](http://consumer.huawei.com/en/press/news/2017/hw-u-035103/)
+ * 2016: 139.3 million units shipped (+30.2% year-on-year)  [(source: Huawei)](http://consumer.huawei.com/en/press/news/2017/hw-u-035103/)
+ * 2017H1: 73.01 million units shipped (+20.6% year-on-year) [(source: Huawei)](http://www.huawei.com/za/news/za/huawei-consumer-business-group-announces-2017-h1-business-results)
+ 
+> _"By the third quarter of 2016, 60% of Huawei’s smartphone shipments were mid- and high-priced devices, a striking reversal of its past reliance on cheap handsets."_
```

Comment:
Don't add quote signs when using `>` as they will be added by the css. Also remember that quotes a flown outside of the regular text so you might too have a shorter version in the quote and a longer version left here. Also don't emphasize full sentences.